doc/classes/StreamPeer.xml

```diff
@@ -81,10 +81,10 @@
 		<method name="get_string">
 			<return type="String">
 			</return>
-			<argument index="0" name="bytes" type="int">
+			<argument index="0" name="bytes" type="int" default="-1">
 			</argument>
 			<description>
-				Get a string with byte-length "bytes" from the stream.
+				Get a string with byte-length [code]bytes[/code] from the stream. If [code]bytes[/code] is negative (default) the length will be read from the stream using the reverse process of [code]put_string[/code].
```

Comment:
You can use `[method put_string]` to make it an hyperlink.

Reply:
Right, must be the the side effect of late night coding :sleeping: .
It also seems that `put_string` is not actually a function in `StreamPeer` which only has `put_utf8_string`.
I guess it should be implemented. I'll give it a try.